### ISSUE TITLE
Directory Picker

### ISSUE THREAD

**Issue by donolsch** [flet-dev/examples#95]
Hi there,
Flet looks really promising! Great work!
I looked through your documentation but couldn't find a widget that supports the selection of multiple folders. It would be extremely handy to have a widget like file picker but instead of selecting files I need to be able to select folders (directories) that are returned as a list. Any plans of implementing that?

Thank you

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/filepicker#get_directory_path ?

**Comment by donolsch** [comment]
Yeah, that works great for one folder. But I can't choose multiple folders. For my app it's essential for the user to be able to select multiple folders. The result should be returned as a list containing the directory paths of the selected folders. Or does this already exist? Am I missing something?

For Tkinter apps I was able to use tkfilebrowser which allowed for multiple folder selection.
https://pypi.org/project/tkfilebrowser/ -> askopendirnames -> allow the selection of multiple folders

**Comment by ndonkoHenri** [comment]
Ah, alright. It was added some months ago (https://github.com/flutter/packages/pull/3871) by the flutter team.
